mrtarget/common/LookupHelpers.py

```diff
@@ -139,7 +137,6 @@ def _get_hpo(self):
         obj = OntologyClassReader()
         hpo_uri = Config.ONTOLOGY_CONFIG.get('uris', 'hpo')
         opentargets_ontologyutils.hpo.get_hpo(obj, hpo_uri)
-        obj.rdf_graph = None
```

Comment:
I think this line is important in saving memory by free objects that should no longer be used for garbage collection.

Reply:
Done!